src/actions/sm-data.js

```diff
@@ -1,4 +1,12 @@
 import * as types from './types';
+import { updateStructureStatus } from './forms';
+
+export function reBuildSMUI(items) {
+  return dispatch => {
+    dispatch(buildSMUI(items));
+    dispatch(updateStructureStatus(0));
```

Comment:
Is there a comment block somewhere which details what `code` values mean for this function?    For example, if I were looking at this for the first time, I'd assume `updateStructureStatus(0)` is sending a `false` argument, `0`.

Reply:
No it does not have any details. Yeah documenting that is a good idea :+1: I will add a comment in actions explaining the code.